scripts/deps.py

```diff
@@ -1,24 +1,31 @@
 import sys
+import jinja2
+import yaml
 
-from conda_build import api
 
-section = {
-    'build': 'requirements/build',
-    'run':   'requirements/run',
-    'test':  'test/requires',
-}
-
-names = set(sys.argv[1:])
+def load_setup_py_data():
+    import os
+    import setuptools
+    os.environ['CONDA_BUILD_STATE'] = 'RENDER'
+    data = {}
 
-if not all(name in section for name in names):
-    print("Got unknown section name. Valid section names are: build, run, or test")
-    sys.exit(1)
+    def _setup(**kw): data.update(kw)
+    setuptools.setup = _setup
+    return data
 
-metadata, _, _ = api.render('conda.recipe')
+meta_src = jinja2.Template(open("conda.recipe/meta.yaml").read())
+meta_src = yaml.load(meta_src.render(load_setup_py_data=load_setup_py_data))
 
+section = {
+    "['build']": meta_src["requirements"]["build"],
+    "['run']":   meta_src["requirements"]["run"],
+    "['test']":  meta_src["test"]["requires"],
+}
+spec = section[str(sys.argv[1:])]
```

Comment:
As a side not it's generally not safe to rely on the string representation to index like this, because it may differe by python version, or platform. `str(["foo"])` might return `"['foo']"` in some cases and `'["foo"]'` in others, making use as a key problematic.